odoo/addons/test_search_panel/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import models
+from . import tests
```

Comment:
Don't import `tests`.  It is done automatically by `--test-enable`.

Reply:
ok